src/views/token-swap/components/quote-selector/quote-selector.view.jsx

```diff
@@ -17,7 +17,6 @@ function QuoteSelector({
       <div className={classes.header}>
         <p className={classes.nameHeader}>Liquidity provider</p>
         <p className={classes.toTokensHeader}>You get (ETH)</p>
-        <p className={classes.rewardHeader}>Reward</p>
```

Comment:
Token swap use different rewards. They are not related to this issue

Reply:
Thx! Reverted!